automated/linux/ima/ima.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+    name: ima-smoke
+    format: "Lava-Test Test Definition 1.0"
+    description: "Check if IMA is enabled in the kernel
+		  The test only checks kernel configs presence,
+          IMA initialization and presence of measurements
+          file. There is no comparison between measurements
+          taken in different boot stages. There is also no
+          requirement to use TPM."
+    maintainer:
+        - milosz.wasilewski@foundries.io 
+    os:
+        - debian
+        - ubuntu
+        - fedora
+        - centos
+    scope:
+        - functional
+    environment:
+        - lava-test-shell
+    devices:
+        - hi6220-hikey
+        - apq8016-sbc
+        - d05
+        - imx8mmevk
+        - imx6ullevk
+
+run:
+    steps:
+        - cd ./automated/linux/ima/
+        - ./ima.sh -c
```

Comment:
./ima.sh -c 
Do we need this "-c" here ?
May be yes. When you have configs parameterized !

Reply:
oups, I thought I removed it :)